packages/x-charts-pro/src/tests/materialVersion.test.tsx

```diff
@@ -2,4 +2,8 @@ import materialPackageJson from '@mui/material/package.json';
 import { checkMaterialVersion } from 'test/utils/checkMaterialVersion';
 import packageJson from '../../package.json';
 
-checkMaterialVersion({ packageJson, materialPackageJson });
+checkMaterialVersion({
+  packageJson,
+  materialPackageJson,
+  packageDirectory: 'packages/x-charts-pro',
```

Comment:
Can't we use something like `import.meta.url` like we use in the vitest configs? Passing the current package path can be error prone, like if we use `packages/x-data-grid` on a charts package we will get the wrong results https://github.com/mui/mui-x/blob/master/vitest.config.mts#L5

Reply:
Good idea. 👍 
Updated the solution.
Let me know what you think. 🙏